core/src/main/java/org/libreoffice/lots/sidebar/WollMuxSidebarContent.java

```diff
@@ -253,8 +253,8 @@ public void windowResized(WindowEvent e)
 
       if (WollMuxSingleton.getInstance().isNoConfig())
       {
-        String text = L.m("No WollMux Configuration found.\n"
-            + "Please setup the WollMux Configuration\n"
+        String text = L.m("No LOTS Configuration found.\n"
```

Comment:
LOTS is a technical abbreviation and should not be used in user visible strings.
Please use the full name here.